### DISCUSSION TITLE
How does the "Exclude from Recommendations work"?

### DISCUSSION THREAD

**Discussion by samuelawachie** [SpotX-Official/SpotX#168] [Question]
If I were streaming from my phone, but using the desktop app (via Connect) to "control" playback and use the function "Exclude this playlist from Recommendations", will it still work on my account as a whole? Or is the functionality only successful if I were streaming directly from the Desktop app with my computer?

**Comment by amd64fox** [comment]
Good question.

As far as I understand, `Exclude playlist from recommendations` feature cannot be synchronized between devices, that is, if you excluded playlist №1 in your desktop application, then the same playlist, but in the mobile application will still not be excluded and it will also be needed to exclude again, but in the mobile application.

It follows that this feature works exclusively on the specific device that is streaming at the moment and does not depend on other devices.

Why they did this is not clear. 
Obviously, suggests that this feature should work by default for all devices at once.

**Reply by samuelawachie** [reply]
I agree that it should carry over to all devices. That should be the more useful configuration of something like that.
I just wonder whether that functionality can be built into the code going forward; and if yes, who are we to petition to?

**Reply by amd64fox** [reply]
Try to create a request on their forum, in my opinion this is the only place.
https://community.spotify.com/
